app/assets/stylesheets/catarse_bootstrap/_extra.sass

```diff
@@ -13,6 +13,8 @@ img, iframe
 .list-answer
   display: none
 
+.project-search-image
+  max-width: 50px !important
```

Comment:
I see, but the fix should remove the !important from img so the CSS would prefer the max-width from the class.
!important is a terrible contagious hack, the more we use it, the more we will have to use it.